connectors/bitbucket-server/src/main/java/com/vmware/connectors/bitbucket/server/BitbucketServerController.java

```diff
@@ -125,14 +125,13 @@ public BitbucketServerController(WebClient rest, CardTextAccessor cardTextAccess
     }
 
     @GetMapping("/test-auth")
-    public Mono<ResponseEntity<Void>> verifyAuth(@RequestHeader(AUTH_HEADER) final String authHeader,
+    public Mono<Void> verifyAuth(@RequestHeader(AUTH_HEADER) final String authHeader,
```

Comment:
Tested ```test-auth``` endpoint of bitbucket connector with valid and invalid credentials. 

For valid credentials, we are sending 200 and 400 in case of invalid credentials.